tests/test_csv.py

```diff
@@ -12,7 +12,7 @@ def test_export_csv_empty(client):
     assert response.status_code == 200
     assert response.mimetype == 'text/csv'
     assert response.is_streamed
-    reader = csv.reader(io.BytesIO(response.data))
+    reader = csv.reader(response.data.decode("utf-8").strip().split("\n"))
```

Comment:
Why not `io.StringIO(response.data)` (or `io.StringIO(response.data.decode('utf8')`) ?

Reply:
ok (: